blob/gcsblob/gcsblob.go

```diff
@@ -134,6 +148,12 @@ const Scheme = "gs"
 //
 //   - access_id: sets Options.GoogleAccessID
 //   - private_key_path: path to read for Options.PrivateKey
+//
+// Currently their use is limited to SignedURL. If left empty they will be guessed
```

Comment:
IIUC, they are only guessed when using the default URLOpener. So, I think that the comments about guessing should be up near line 23, not here.